### ISSUE TITLE
”Efficient Deep CNN-BiLSTM Model for Network Intrusion Detection“

### ISSUE THREAD

**Issue by Bob0123456** [razor08/Efficient-CNN-BiLSTM-for-Network-IDS#1]
Hello, your code is to comprehensively process the training set and test set, and then test on the model. The result of this test set will be inconsistent with the original test set. Is the result obtained in this way accurate?

**Comment by jayxsinha** [comment]
Yes. You can run in the way you suggested, it will still have similar results.
